aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/samples/sample6.cs

```diff
@@ -18,9 +18,7 @@ public void InitializeDatabase(ApplicationDbContext context)
       {
         // query to check if MigrationHistory table is present in the database 
         var migrationHistoryTableExists = ((IObjectContextAdapter)context).ObjectContext.ExecuteStoreQuery<int>(
-        string.Format(
-          "SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = '{0}' AND table_name = '__MigrationHistory'",
-          "[Insert your database schema here - such as 'users']"));
+          "SELECT COUNT(*) FROM information_schema.tables WHERE table_schema = 'IdentityMySQLDatabase' AND table_name = '__MigrationHistory'");
```

Comment:
Can this not use the origin pattern but change the wording to say "Insert your database schema (commonly known as the database name) here - such as 'users'"?

Reply:
I'd prefer to bring the sample inline with the tutorial. The dB has a known name in the walk-through, so there's no need to leave the reader guessing what the value should be in this case. It's also obvious from the name what it's referring to.

Reply:
![image5](https://user-images.githubusercontent.com/1622880/30829611-e7d94814-a206-11e7-8a42-a28b95cc5f0b.png)